updateEmoji.js

```diff
@@ -1,4 +1,4 @@
-var rawEmoji = require('./config/emoji_pretty.json');
```

Comment:
I think you'll want to also delete `config/emoji_pretty.json` from the repo now that it's provided by an npm dependency instead.